### ISSUE TITLE
Add Miro Support

### ISSUE THREAD

**Issue by wobintosh2** [podlove/podlove-subscribe-button#79]
Hej guys,

would be nice, if you could add Miro to the Subcscribe-Button :)

Subscribe-Buttons for Miro are existing: http://subscribe.getmiro.com/

**Comment by timpritlove** [comment]
Interesting: essentially the backend creates an OPML file containing the feed and delivers it with a MIME type of "application/x-miro" and a filename with a ".miro" extension. This is supposed to activate the Miro application which will open the file at launch and read the feed from the OPML structure.

However, this is a mechanism than our PSB currently uses. We rely on activiation via URL schemes (and probably per-client registered URLs in the future). However, we have talked about adding OPML support to enable builk subscription as OPML can potentially list an unlimited number of feeds to be subscribed to.

 While creating an OPML file is possible, not all clients define a dedicated MIME-Type/Extension as Miro does. While I would consider this a solid solution it is not common. A dedicated URL scheme in the form of something like `<client-scheme>://subscribe-via-opml/<opml-url>` (analogous to `<client-scheme>://subscribe/<feed-url>`) would be the best but we would need a huge campaign to make this happen.

To summarize: this path could be chosen once we tackle OPML in general but it would be so much easier if Miro would just add `miro:` protocol scheme. Unfortunately, this app is no longer being developed so this will probably not happen anytime soon.
